README.md

```diff
@@ -14,22 +15,20 @@ See benchmark results [here](https://intelpython.github.io/scikit-learn_bench).
 * [Running Python benchmarks with runner script](#running-python-benchmarks-with-runner-script)
 * [Supported algorithms](#supported-algorithms)
 * [Algorithms parameters](#algorithms-parameters)
-* [Legacy automatic building and running](#legacy-automatic-building-and-running)
 
 ## Prerequisites
 - `python` and `scikit-learn` to run python versions
 - pandas when using its DataFrame as input data format
-- `icc`, `ifort`, `mkl`, `daal` to compile and run native benchmarks
 - machine learning frameworks, that you want to test. Check [this item](#how-to-create-conda-environment-for-benchmarking) to get additional information how to set environment.
 
 ## How to create conda environment for benchmarking
 
 Create a suitable conda environment for each framework to test. Each item in the list below links to instructions to create an appropriate conda environment for the framework.
 
-* [**scikit-learn**](https://github.com/PivovarA/scikit-learn_bench/blob/master/sklearn/README.md#how-to-create-conda-environment-for-benchmarking)
-* [**daal4py**](https://github.com/PivovarA/scikit-learn_bench/blob/master/daal4py/README.md#how-to-create-conda-environment-for-benchmarking)
-* [**cuml**](https://github.com/PivovarA/scikit-learn_bench/blob/master/cuml/README.md#how-to-create-conda-environment-for-benchmarking)
-* [**xgboost**](https://github.com/PivovarA/scikit-learn_bench/tree/master/xgboost/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**scikit-learn**](https://github.com/IntelPython/scikit-learn_bench/blob/master/sklearn/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**daal4py**](https://github.com/IntelPython/scikit-learn_bench/blob/master/daal4py/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**cuml**](https://github.com/IntelPython/scikit-learn_bench/blob/master/cuml/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**xgboost**](https://github.com/IntelPython/scikit-learn_bench/tree/master/xgboost/README.md#how-to-create-conda-environment-for-benchmarking)
 
 ## How to enable daal4py patching for scikit-learn benchmarks
 Set specific environment variable `export FORCE_DAAL4PY_SKLEARN=YES`
```

Comment:
Patching must be by default. And with explicit patch_sklearn() function

Reply:
Are you sure this is what we want? If we want to measure the stock, we need to add something like `export SKLEARN_RUN=1`. Which looks strange. As I understand it, we would like to expand the Scikit-Learn functions without daal for example TSNE, SVR, etc.

Reply:
Probably we, need to have options for benchmarks run:
```
python runner.py --sklearn_version [stock | intel-optimized]
```
and by default it's ```intel-optimized```

I really don't like some env variables - we will have a lot of issues with this.

Reply:
Ok. I don't like that, but I did. See if it worked out ok?